.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
@Eilon @muratg, do you want this in all of our repos? We shouldn't put it in just one, there's nothing special about this repo.